builder/assets/kinetic-rosdep-clever.yaml

```diff
@@ -529,6 +529,12 @@ libogg:
 vl53l1x:
   debian:
     stretch: [ros-kinetic-vl53l1x]
+ws281x:
```

Comment:
Why not put this at the end of the file?

Reply:
Just to kinda group peripheral drivers together (put near vl53l1x driver).